frontend/lib/laletterbuilder/letter-builder/my-letters.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { ProgressStepProps } from "../../progress/progress-step-route";
+import Page from "../../ui/page";
+import { LaLetterBuilderRouteInfo } from "../route-info";
+import { li18n } from "../../i18n-lingui";
+import { t } from "@lingui/macro";
+
+export const LaLetterBuilderMyLetters: React.FC<ProgressStepProps> = (
+  props
+) => {
+  return (
+    <Page title="My letters" withHeading="big" className="content">
```

Comment:
quick note to internationalize the page title (eventually)

Reply:
good catch ty!